app/controllers/services_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ServicesController < ApplicationController
+  def index
+    @services = Service.all
+  end
+
+  def new
+    @service = Service.new
+  end
+
+  def create
+    @service = Service.new(:service_params)
+    @service.save
+  end
+
+  def update
+    @service = Service.find(params[:id])
```

Comment:
@wiliajc87  With the update method, doesn't there need to be a line updating the params?